tests/test_interpreter.py

```diff
@@ -65,6 +65,10 @@ def size(Node(l, r)) = size(l) + size(r)
 size(Node(Empty(), Leaf(10))) == 1
 '''
 
+# Other examples: http://coconut.readthedocs.io/en/master/DOCS.html#usage
+COMPILE_ARGS = '--line-numbers'
+COMPILE_ARGS_OUTPUT = b'(print)(\\"hello, world!\\")  # line 1'
```

Comment:
Possibly add test cases with `async def` for python2 and python3.

Reply:
Done